platform-tests/src/test/java/org/junit/platform/commons/util/ExceptionUtilsTests.java

```diff
@@ -64,15 +67,16 @@ void readStackTraceForLocalJUnitException() {
 		}
 	}
 
-	@Test
-	void pruneStackTraceOfCallsFromSpecificPackage() {
+	@ParameterizedTest
+	@ValueSource(strings = { "org.junit.", "jdk.internal.reflect.", "sun.reflect." })
+	void pruneStackTraceOfCallsFromSpecificPackage(String shouldBePruned) {
```

Comment:
Is there a concrete need to throw and catch these exceptions?

If not, just rely on the fact that `fillInStackTrace()` is called by the exceptions constructor (technically in `java.lang.Throwable.Throwable(String)`).